extensions/typescript-language-features/src/languageFeatures/tsconfig.ts

```diff
@@ -110,6 +114,55 @@ class TsconfigLinkProvider implements vscode.DocumentLinkProvider {
 	}
 }
 
+// Reference: https://github.com/microsoft/TypeScript/blob/febfd442cdba343771f478cf433b0892f213ad2f/src/compiler/commandLineParser.ts#L3005
+/**
+ * @returns Returns undefined in case of lack of result while trying to resolve from node_modules
+ */
+const getTsconfigPath = async (baseDirUri: vscode.Uri, extendsValue: string): Promise<vscode.Uri | undefined> => {
+	// Don't take into account a case, where tsconfig might be resolved from the root (see the reference)
+	// e.g. C:/projects/shared-tsconfig/tsconfig.json (note that C: prefix is optional)
+
+	const isRelativePath = ['./', '../'].some(str => extendsValue.startsWith(str));
+	if (isRelativePath) {
+		const absolutePath = vscode.Uri.joinPath(baseDirUri, extendsValue);
+		if (await exists(absolutePath)) { return absolutePath; }
+		// Will suggest to create a .json variant if it doesn't exist yet
+		return absolutePath.with({
```

Comment:
For a path `./tsconfig.base.json` that doesn't exist on disk, I believe this will end up trying to create a file called `tsconfig.base.json.json`. Instead you should only add `.json` when the path does not already have a file extension

Reply:
You was right! Fixed. I should have better looked at the reference.